doc/api/url.md

```diff
@@ -389,6 +389,46 @@ console.log(myURL.href);
 
 Invalid URL protocol values assigned to the `protocol` property are ignored.
 
+##### Special Schemes
+
+The WHATWG URL Standard considers a handful of URL protocol schemes to be
```

Comment:
Linking the text `WHATWG URL Standard` to the standard itself would be useful here. It's already linked in two other places in the doc, so it's just a matter of changing the markdown to `[WHATWG URL Standard][]`.